spec/helpers/comments_helper_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe CommentsHelper do
+  describe "#comment_path" do
+    let(:comment) { VCR.use_cassette('planningalerts') { create(:confirmed_comment) } }
+
+    it "returns the path for the application with an anchor for the comment" do
+      expect(helper.comment_path(comment)).to eq application_path(comment.application, anchor: "comment#{comment.id}")
```

Comment:
The reason I used the path helper here was to isolate this test from any changing to the urls made. Rails generates that part, so it feels like testing the framework to hard code in the string. The test output it is still the full string so you can see exactly what’s happening:

```
expected: "/applications/1#commet1"
got: "/applications/1#comment1"
```

I think this is still testing quite tightly what we want to test. What do you think @henare ? This is an interesting issue.